python.html.markdown

```diff
@@ -107,26 +115,36 @@ not False  # => True
 
 # Strings can be added too!
 "Hello " + "world!"  # => "Hello world!"
+
 # Strings can be added without using '+'
 "Hello " "world!"  # => "Hello world!"
 
 # ... or multiplied
 "Hello" * 3  # => "HelloHelloHello"
 
+# A number can be a string
+type("2") # => <type 'str'>
```

Comment:
I think mentioning this make things more confusing. Remember that the reader is assumed to know at least one other programming language -- they probably understand that a string of number characters is considered a string.

Probably more useful to demonstrate that you can't just `+` a number onto a string for the benefits of php and javascript devs.